webpack.config.js

```diff
@@ -40,14 +41,77 @@ function getDefaultConfiguration() {
 };
 }
 
+function webpackForTest(testFileName) {
+  var match = testFileName.match(/\/?([^\/]*)-test.ts$/);
```

Comment:
ew `const` instead of `var` please

Reply:
That's what I get for reusing old code 😄